modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/DataEngineLayoutBuilderHandler.es.js

```diff
@@ -124,11 +124,11 @@ export default function DataEngineLayoutBuilderHandler({namespace}) {
 		if (
 			isElementInnerSelector(
 				target,
+				'.cke_dialog',
 				'.ddm-form-builder-wrapper',
 				'.multi-panel-sidebar',
 				'.lfr-icon-menu-open',
-				'.input-localized-content',
-				'.cke_dialog'
+				'.input-localized-content'
```

Comment:
Hi @rebsilva, just a question. Do you think this parameters should also be sorted?

Reply:
Sure Aline! I forgot to sort them, thanks for telling me!